src/Widget/WidgetInterface.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Widget;
+
+use Symfony\Component\HttpFoundation\Response;
+
+interface WidgetInterface
```

Comment:
most of setters are not needed in this interface

Reply:
Removed most, and added `__invoke`. I've done this because otherwise PHPstorm and PHPStan would complain about this: 

![Schermafbeelding 2019-04-19 om 15 19 34](https://user-images.githubusercontent.com/1833361/56425937-8ebf9180-62b6-11e9-93d6-509ead1d6f37.png)